spring-xd-spark-streaming-tests/src/test/java/org/springframework/xd/spark/streaming/RabbitTransportSparkStreamingTests.java

```diff
@@ -47,9 +47,8 @@ protected void createStream(String streamName, String stream) {
 		addQueueNames(streamName, stream);
 	}
 
-	@After
-	public void cleanupRabbitQueues() {
-		super.tearDown();
+	@AfterClass
```

Comment:
This is just the rabbit queues that we want to delete after the test run (as we don't normally delete the rabbit queues as part of stream destroy). Since we destroy the streams between the tests, I think we are good here... Moreover, I had an issue (one of the queues wasn't cleaned up) running the cleanup of rabbit queues between the tests.